web/.eslintrc.js

```diff
@@ -75,7 +75,7 @@ module.exports = {
     'no-use-before-define': ['error', { variables: true, functions: false, classes: true }],
     'prefer-const': 'error',
     'prefer-template': 'error',
-    'react/jsx-filename-extension': ['error', { 'extensions': ['.js', '.jsx'] }],
+    'react/jsx-filename-extension': ['error'],
```

Comment:
I updated this rule to ensure we enforce correct file extensions going forward :)